pyramid_hypernova/batch.py

```diff
@@ -152,7 +153,7 @@ def submit(self):
             synchronous = len(job_groups) == 1
 
             for job_group in job_groups:
-                request_headers = self.plugin_controller.transform_request_headers({})
+                request_headers = self.plugin_controller.transform_request_headers({}, self.pyramid_request)
```

Comment:
why pass it in? this is the same object, can't it just use `self.pyramid_request`?

Reply:
It looks like plugin_controller is a [separate object](https://github.com/Yelp/pyramid-hypernova/blob/1487a157a9bbf99b89c1486ee0abd6e9c7a5dc2c/pyramid_hypernova/plugins.py#L6) that gets constructed without knowledge of the pyramid request. Is there another way of accessing the pyramid request or something inside the object?

Reply:
oh you're right, i misread the code (saw a call to `self....()` and assumed it was a function on the same object). ignore this.